index.bs

```diff
@@ -681,8 +701,10 @@ When this method is invoked, the user agent MUST execute the following algorithm
         [=hash of the serialized client data=]
     : {{tokenBinding}}
     :: The [=Token Binding ID=] associated with |callerOrigin|, if one is available.
-    : {{CollectedClientData/extensions}}
+    : {{CollectedClientData/clientExtensions}}
     :: |clientExtensions|
```

Comment:
Since [`AuthenticationExtensions`](https://api.csswg.org/bikeshed/?url=https://raw.githubusercontent.com/selfissued/webauthn/mbj-extensions-connective-tissue/index.bs#dictdef-authenticationextensions) doesn't have any fields, it doesn't quite work to initialize it from a map. Either, we should add [partial dictionary](https://heycam.github.io/webidl/#dfn-partial-dictionary) definitions for each extension, or maybe it should be a [record](https://heycam.github.io/webidl/#idl-record) instead?